python/orca/example/learn/horovod/pytorch_estimator.py

```diff
@@ -51,8 +51,8 @@ def optimizer_creator(model, config):
 
 def scheduler_creator(optimizer, config):
     """Returns a learning rate scheduler wrapping the optimizer.
-    You will need to set ``TorchTrainer(scheduler_step_freq="epoch")``
-    for the scheduler to be incremented correctly.
+    By default a scheduler will increment automatically every epoch,
+    and you can update the scheduler at the right time through the hooks.
```

Comment:
you can customize the behavior of the scheduler through the MainHook?

Reply:
> you can customize the behavior of the scheduler through the MainHook?

Yes, by overriding `on_lr_adjust` and other hooks.For example if you want to step by iteration:
```python
def on_lr_adjust(self, runner):
      pass

def after_train_iter(self, runner):
      runner.scheduler.step()
```

Reply:
I mean modify the doc... not explain to me...😂

Reply:
> I mean modify the doc... not explain to me...😂

All right😂then as I mentioned in the description I will add this later, together with some other typical usage cases.